ScenarioFiles.xml

```diff
@@ -850,15 +848,15 @@
           <Squawk Code="1006" ModeC="true" />
           <Cleared Approach="true" />
           <Position Latitude="-26.78138889" Longitude="152.55861111" Speed="250" Altitude="4000" />
-          <FlightPlan Active="true" Departure="YBMK" Destination="YBBN" Alternative="" Rules="I" CruiseSpeed="428" ETD="0000" ATD="0000" RFL="32000" EETHours="00" EETMins="00" FuelHours="0" FuelMins="0" Remarks="/v/">DCT MK W889 RK V307 SUSGI DCT SMOKA SMOK9A/19R</FlightPlan>
-          <Route>SMOKA RWY19R</Route>
+          <FlightPlan Active="true" Departure="YBMK" Destination="YBBN" Alternative="" Rules="I" CruiseSpeed="428" ETD="0000" ATD="0000" RFL="32000" EETHours="00" EETMins="00" FuelHours="0" FuelMins="0" Remarks="/v/">DCT MK W889 RK V307 SUSGI DCT SMOKA OTGAT GARTH BURPA IGBON PAMBU ATRAX LAVIN OSOKO YBBN/19R</FlightPlan>
```

Comment:
These STAR issues have been fixed here: https://github.com/vatpac-technology/simulator-scenarios/pull/111

No need to enter in all the STAR waypoints, the flight plan just had the wrong STAR number